sunpy/net/tests/test_attr.py

```diff
@@ -365,7 +376,7 @@ def test_asterisk_attrs(ALL):
 ])
 def test_single_pair_argument_attrs(wrong_name):
     # This checks that other single string entries fail.
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match=re.escape("Invalid value given for * registration: [('not star',)].") + "|" + re.escape("Invalid value given for * registration: [('*whoops',)].")):
```

Comment:
Just check for the "Invalid value given for"?

Reply:
Done. Thank You.